### DISCUSSION TITLE
Why --ignore-dot does not ignore .rgignore?

### DISCUSSION THREAD

**Discussion by balazser** [BurntSushi/ripgrep#2197] [General] [answered]
Why `--ignore-dot` switch ignores `.ignore` but not `.rgignore`? Although if `.rgignore` file presents in the parent dictionary it can be ignored with `--ignore-parent`.

**Comment by BurntSushi** [comment, marked as answer]
It's usually good to provide a test case so that we're all on the same page. Here's one:

```
$ mkcd /tmp/rg-2197
$ touch a b c
$ echo a > .ignore
$ echo b > .rgignore
$ rg --files
c
$ rg --files --no-ignore-dot
c
a
```

So I suppose the question here is, "why is `b` still ignored?"

Good question. Looks like it might be a bug.

**Reply by BurntSushi** [reply]
I created #2198.

**Reply by balazser** [reply]
Thank you! :pray:
